src/gausskernel/storage/access/k2/session.cpp

```diff
@@ -199,12 +172,114 @@ sh::Response<>  TxnManager::CreateCollection(const std::string& collection_name,
             .writeIOPs = 0,
             .minNodes = 1   // K2 Http proxy hangs if minNodes = 0
         },
-        .retentionPeriod = sh::Duration(1h) * 90 * 24,  //TODO: get this from config or from param in
+        .retentionPeriod = cconf.getDurationMillis("retention_period", 1h*24*90),
```

Comment:
should the 90 here be a 60?

Reply:
this is the same as what we had before - 90 days (1h * 24 * 90)